src/en/nineanime/build.gradle

```diff
@@ -3,9 +3,9 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlinx-serialization'
 
 ext {
-    extName = '9anime'
-    pkgNameSuffix = 'en.nineanime'
-    extClass = '.NineAnime'
+    extName = 'Aniwave'
+    pkgNameSuffix = 'en.aniwave'
```

Comment:
this should stay the same so that users can update the app
```suggestion
    pkgNameSuffix = 'en.nineanime'
```

Reply:
yeah my bad. I forgot